lambda_backend/route_microservice/src/reportRoute.ts

```diff
@@ -40,22 +41,27 @@ const reportRoute: Handler = async (event: ReportRouteEvent) => {
         ':reports': reports as AttributeValue,
       },
     };
+    logger.info('reportRoute updateItem', { data: { username } });
     try {
       await dynamoDb.update(updateItemInput).promise();
     } catch (error) {
+      logger.error('reportRoute error', { data: { username, error: error.stack } });
       throw createError(500, 'Error updating item', error);
     }
 
     const publishInput: PublishInput = {
       Message: `Report\nBy: ${username}\nOwner: ${routeOwnerUsername}\nAt: ${createdAt}\nTally: ${reports.length}`,
       TopicArn: process.env['TELEGRAM_SNS_ARN'],
     };
+    logger.info('reportRoute publishSNS', { data: { username } });
     try {
       await SNSInstance.publish(publishInput).promise();
     } catch (error) {
+      logger.error('reportRoute error', { data: { username, error: error.stack } });
```

Comment:
Maybe can differentiate with the previous error by changing the log to `reportRoute SNS publish error` or smth